design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -0,0 +1,216 @@
+type Query {
+    estimateShippingCostForNegotiableQuote(
+        cart_id: String!
```

Comment:
Can you help us understanding why a `cart_id` is used here? From a UI dev standpoint it's not very clear

Reply:
We have removed the `cart_id` field as we have an option to get it from the user context. Thanks